addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2052,6 +2052,36 @@ const Wysiwyg = Widget.extend({
             : !finalOptions.snippets;
 
         return finalOptions;
+    },
+     _getContentEditableAreas: function () {
+        const editableNodes = this.options.getContentEditableAreas ? this.options.getContentEditableAreas() : [];
+        return [...editableNodes].concat(
+            $(".o_we_banner_container .o_we_banner_editable").toArray()
+        );
+    },
+    _getReadOnlyAreas: function () {
+        const readOnlyNodes = this.options.getReadOnlyAreas ? this.options.getReadOnlyAreas() : [];
+        return [...readOnlyNodes].concat(
+            $(".o_we_banner_container").toArray()
+        );
+    },
```

Comment:
Is there a particular reason you didn't use the classes `o_editable` and `o_not_editable` instead ? I feel like they would be enough for your use case.

Reply:
does not seem to work. I cannot even enter in the banner to type something. (the text cursor enters it then directly goes out)
```
 <div t-attf-class="o_not_editable lh-1 d-flex alert alert-#{alertClass}" role="status" data-oe-protected="true">
            <i t-attf-class="fa #{iconClass}" t-att-aria-label="label"/>
            <div class="o_editable w-100 ms-3 oe_unremovable" data-oe-protected="false">
                <p><br/></p>
            </div>
        </div>
```

If I still do the following 
```
<div t-attf-class="o_not_editable lh-1 d-flex alert alert-#{alertClass}" role="status" data-oe-protected="true" contenteditable="false">
            <i t-attf-class="oe_unremovable fa #{iconClass}" t-att-aria-label="label"/>
            <div class="o_editable w-100 ms-3 oe_unremovable" data-oe-protected="false"
                t-att-contenteditable="!readonly ? 'true' : 'false'">
                <p><br/></p>
            </div>
        </div>
```
it works when adding the banner to the pad (as contenteditable is already set) but after reloading the record, it does not re-add the contenteditable property on the Elements. As I understand it, o_not_editable makes the job (contenteditable False is correctly added to the Element) but not o_editable.
Is there something I should know

Reply:
You are right. I think it _should_ work, so I made a commit to make it work. Could you try with https://github.com/odoo-dev/odoo/commit/c69eb80d7b4aca2760bb239c76be7878f3e79559 ?

Reply:
Working perfectly ! Thanks ! :heart: 
I'll ask DBO and LNA to test this.
Regarding tests, what do you suggest ? I'dont really know I can test this easily. I thought of a tour but web-editor seems not a good place to put it. Maybe In knowledge, but that's not a knowledge feature so.. meh'